.github/workflows/regression-tests.yml

```diff
@@ -336,7 +336,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
+        sys:
+          - { icon: '⬛', msys: MINGW32 }
+          - { icon: '🟦', msys: MINGW64 }
+          - { icon: '🟨', msys: UCRT64  } # Experimental!
```

Comment:
Is this something we want support? I think this probably needs wider discussion in an issue first. We have been trying to reduce the amount of things we support, not extend it.

Reply:
I think that UCRT64 might replace MINGW64 in the near future, since it makes it easier to use DLLs from MSVC in MSYS2 or the other way round. I believe that is particularly interesting for cocotb, since it uses Python and it builds shared libraries. Hence, I would recommend to test both MINGW64 and UCRT64 for now.

With regard to 32 bits, that's up to you. I know some people use GHDL's 32 bit version because they want the `mcode` backend. However, that should be supported on 64 bits sometime in the future. It is acceptable to deprecate 32 bits.

Reply:
See https://www.msys2.org/docs/environments/